addons/l10n_se/data/account.account.template.csv

```diff
@@ -220,27 +214,24 @@
 "a5410","5410","Förbrukningsinventarier","account.data_account_type_expenses","l10nse_chart_template","False"
 "a5420","5420","Programvaror","account.data_account_type_expenses","l10nse_chart_template","False"
 "a5460","5460","Förbrukningsmaterial","account.data_account_type_expenses","l10nse_chart_template","False"
-"a5480","5480","Arbetskläder","account.data_account_type_expenses","l10nse_chart_template","False"
```

Comment:
This is not listed as the recommended minimum. However, it's included in the complete BAS 2020 and should not be removed.

Reply:
There is no logic of adding just some of the accounts in the extended list.

Reply:
Not in minimum ( https://github.com/odoo/odoo/pull/53092#discussion_r454259845 )